terraform/environments/srdg/main.tf

```diff
@@ -20,6 +20,8 @@ module "common" {
 
   create_vpc = true
 
+  availability_zone = "ap-northeast-2c"
```

Comment:
Should this be added to https://github.com/planetarium/9c-infra/tree/main/terraform/environments/main instead of srdg?

Reply:
By default, we use every availability_zones in the region for creating subnets.
this variable seems to be used only for nat gateway, which i recommend to specify not to be confused with this variable😕

Reply:
@area363 it is intended to be srdg, while default value for the variable is us-east-2, only srdg's az is required to be changed.
@sky1045 i agree for that. would another variable name such as nat_availability_zone be better?